docs/docs/cmd/login.mdx

```diff
@@ -44,6 +44,9 @@ m365 login [options]
 
 `--cloud [cloud]`
 : Cloud to connect to. Allowed values `Public`, `USGov`, `USGovHigh`, `USGovDoD` and `China`. Default `Public`
+
+`--ensure [ensure]`
```

Comment:
Since `--ensure` is a flag and doesn't expect a value, let's remove `[ensure]` to stick to our convention